src/dcurl.c

```diff
@@ -33,9 +33,21 @@
 
 /* check whether dcurl is initialized */
 static bool isInitialized = false;
+static uint8_t initStatus = 0;
```

Comment:
How about renaming from `initStatus` to `runtime_caps` or `caps`?

Reply:
I prefer `runtimeCaps`.

It reminds me to unify the variable naming with **Camel-Case** or others.

Reply:
Done.